router/router.go

```diff
@@ -266,8 +269,9 @@ func (r *Router) Events(forwarder router.RouterV1_EventsServer) error {
 				continue
 			}
 			log := fwdlog.WithFields(logrus.Fields{
-				"gw_network_id": gatewayNetworkID,
-				"gw_owner":      gatewayOwner,
+				"gateway_id":         gatewayID,
+				"gateway_network_id": gatewayNetworkID,
```

Comment:
We use `gw_network_id` everywhere else, lets keep it the same in the entire codebase. This happens multiple times in this file.